rules_python/piptool.py

```diff
@@ -138,8 +138,10 @@ def pip_install():
 all_requirements = _requirements.values()
 
 def requirement(name):
-  name = name.replace("-", "_").lower()
-  return _requirements[name]
+  name_key = name.replace("-", "_").lower()
+  if name_key not in _requirements:
+    fail("Could not find pip-provided dependency: '%s'" % name)
```

Comment:
Maybe it would be interesting to include what's available?

Reply:
We could, but since that list is unbounded, I'm concerned that it would be too much for this error string.

Reply:
This is Python, not Node.js ;-)

SGTM